egs/swbd/s5/local/rnnlm/run_lstm_d.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# Copyright 2012  Johns Hopkins University (author: Daniel Povey)  Tony Robinson
+#           2015  Guoguo Chen
+#           2017  Hainan Xu
+
+# This script trains LMs on the swbd LM-training data.
+# This script takes no command-line arguments but takes the --cmd option.
+
+# Begin configuration section.
+cmd=run.pl
```

Comment:
this is not right.  You should be sourcing cmd.sh and using e.g. train_cmd from there.
But I notice later in the script you are not even using this, you use queue.pl directly.